src/main/java/karuna/karuna_backend/content/domain/ContentService.java

```diff
@@ -36,4 +40,24 @@ private ContentDTO convertToDTO(HashMap<String, String> contents) {
         ContentDTO contentDTO = new ContentDTO(contents);
         return contentDTO;
     }
-}
+
+    public ContentResponseDTO addOrUpdateContent(ContentCreateDTO contentCreateDTO) {
+        Optional<Content> contentOptional = contentRepository.findByPageAndKey(contentCreateDTO.page(), contentCreateDTO.key());
+        AtomicReference<Content> contentAtomic = new AtomicReference<>(null);
+        contentOptional.ifPresentOrElse(
+                target -> {
+                    target.setValuePl(contentCreateDTO.value());
+                    contentAtomic.set(contentRepository.save(target));
+                },
+                () -> {
+                    Content newContent = Content.builder()
+                            .page(contentCreateDTO.page())
+                            .key(contentCreateDTO.key())
+                            .valuePl(contentCreateDTO.value())
+                            .build();
+                    contentAtomic.set(contentRepository.save(newContent));
+                });
+        Content content = contentAtomic.get();
+        return new ContentResponseDTO(content.getId(), content.getPage(), content.getKey(), content.getValuePl());
```

Comment:
Mayby move this line and upper to private method with atomicreference parameter?